clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinatorTest.java

```diff
@@ -537,6 +538,7 @@ public void testResetGeneration() {
         coordinator.ensurePartitionAssignment();
 
         // now switch to manual assignment
+        client.prepareResponse(new LeaveGroupResponse(Errors.NONE.code()).toStruct());
```

Comment:
I was surprised this was the only addition to the tests. It seems we're missing a `testClose()`? That is by far the much more common scenario (`resetGeneration` looks like it is only used as a result of unsubscribing from all topics).